Dockerfile

```
@@ -10,25 +13,18 @@ RUN sudo apk -U upgrade --no-cache && sudo apk add --no-cache \
     oniguruma-dev \
     openssl-dev
 
-# Use Opam 2.2 and enable the backup mirror if primary sources of packages are unavailable
-RUN sudo mv /usr/bin/opam-2.2 /usr/bin/opam && opam update
-RUN opam option --global 'archive-mirrors+="https://opam.ocaml.org/cache"'
-
-# Branch freeze was opam-repo HEAD at the time of commit
-RUN cd ~/opam-repository && git reset --hard 584630e7a7e27e3cf56158696a3fe94623a0cf4f && opam update
-
-WORKDIR /home/opam
-
-# Install opam dependencies
-COPY --chown=opam ocamlorg.opam .
-RUN opam install . --deps-only
+RUN curl -sSL https://github.com/ocaml-dune/dune-bin-install/releases/download/v1/install.sh | sh -s 3.19.1 --install-root /usr --no-update-shell-config
```

To avoid the issue with ipv6 and github reported by @cuihtlauac and to install the latest stable version of dune, update it to:
```suggestion
RUN curl -4fsSL https://github.com/ocaml-dune/dune-bin-install/releases/download/v3/install.sh | sh -s 3.20.1 --install-root /usr --no-update-shell-config
```

---

Thank you!